cardano-api/src/Cardano/Api/Query.hs

```diff
@@ -246,6 +250,10 @@ data QueryInShelleyBasedEra era result where
     :: Maybe (Set PoolId)
     -> QueryInShelleyBasedEra era (SerialisedPoolState era)
 
+  QueryPoolDistr
```

Comment:
Let's stick with the current style and name this `QueryPoolDistribution` (as compared to `QueryStakeDistribution`)